Doppler.BillingUser/Infrastructure/PromotionRepository.cs

```diff
@@ -13,14 +13,14 @@ public PromotionRepository(IDatabaseConnectionFactory connectionFactory)
             _connectionFactory = connectionFactory;
         }
 
-        public async Task UpdateTimeToUse(Promotion promocode, string operation)
+        public async Task UpdateTimeToUse(Promotion promocode)
```

Comment:
What do you think about:

```suggestion
        public async Task IncrementUsedTimes(Promotion promocode)
```

Reply:
Yes is more descriptive with this change, done